docs/docker_container.md

```diff
@@ -229,6 +229,7 @@ Configuration options for server are defined only via command line options and d
 | `grpc_workers` | `integer` |  Number of the gRPC server instances (must be from 1 to CPU core count). Default value is 1 and it's optimal for most use cases. Consider setting higher value while expecting heavy load. ||
 | `rest_workers` | `integer` |  Number of HTTP server threads. Effective when `rest_port` > 0. Default value is set based on the number of CPUs. ||
 | `file_system_poll_wait_seconds` | `integer` |  Time interval between config and model versions changes detection in seconds. Default value is 1. Zero value disables changes monitoring. ||
+| `rest_timeout` | `integer` | Rest API request timeout in miliseconds. Default value is 5000. ||
```

Comment:
Providing this timeout in milliseconds seems a little overkill to me? Wouldn't seconds be enough?

Reply:
@mzegla  miliseconds are for compatibility reasons with TFS